testdata/rabbit/webScraping/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"net/http"
+	"os"
+)
+
+func main() {
+	links := readCSV()
+	if links == nil {
+		fmt.Println("Failed to find any links.")
```

Comment:
I tend to use `log.Fatalf` and `log.Printf` in a program like this.  When you want to show a message, then exit the program with an error `log.Fatalf` does that for you.